www/src/index.ts

```diff
@@ -117,7 +120,8 @@ function main() {
         setTimeout(() => {
             map.beginAnimation();
         }, 0.5);
-    });
+    };
+    map.addEventListener(MapViewEventNames.FrameComplete, onFrameComplete);
```

Comment:
this fix is strange, why do we need to use FrameComplete here? can't we just start the animation without waiting for FrameComplete?